### ISSUE TITLE
Future updates

### ISSUE THREAD

**Issue by cryohellinc** [YasserDRIF/Toggle-Tweaker#3]
Good day Yasser,

First of all wanted to say a big Thank You for development of this amazing tool. I use it on each one of my PC's, laptops and even work PC! (shhh don't tell my boss ;) )

Main question is - while windows 10 updates keep rolling out, do You have any plans to release a new version in future?

Kind regards,
Cryohellinc.

**Comment by Drakonas** [comment]
I would like to know this as well. It seems to be broken on current versions of Windows 10.

**Comment by Atavic** [comment]
Maybe you can look @CHEF-KOCH Regtweaks for [Cortana](https://github.com/CHEF-KOCH/regtweaks/tree/master/Win%2010/Cortana) and [Edge](https://github.com/CHEF-KOCH/regtweaks/tree/master/Win%2010/Edge)... But the best way to slim down the OS is to make your own ISO with MSMG Toolkit. I'm using *Windows 10 Pro No Bloatware Edition 64-Bit*, search on *majorgeeks* and click on *Author:* to know more.

**Comment by Hyperyia** [comment]
That's why you should use LTSB guys, don't use normal CB/CBB editions of Windows 10.

**Comment by Hyperyia** [comment]
LOL, it's not a myth, it's a fact that LTSB (LTSC) is superior than CB/CBB. 

First of all: LTSB (LTSC) comes without UWP apps; there is no Cortana, Edge, Store, Solitaire Collection and sponsored third party adware crap like Candy Crush Soda Saga & Minecraft. 

Secondly: LTSB (LTSC) doesn't receive feature updates, so your registry and GPO tweaks will NEVER get resetted by updates, unlike on CB/CBB editions on which every 6 months your settings get resetted, because each "feature update" acts as a system refresh. 

And who even cares about Defender? It's worst antivirus ever, smart people use ESET or Kaspersky and disable Defender completely in gpedit.msc

Everyone uses Windows for Win32 API software, literally nobody cares about UWP apps and those crappy Cloud Computing services, maybe except extreme masochists and Microsoft shills.
My powerful desktop PC is not a smartphone or tablet, im not Touch Screen pleb. I am PowerUser, so i want to store everything locally on my 768 TB 64-bays NAS, don't need Cloud Syncing, OneDrive and other shit. 
I backup everything manually & locally.


>MS even says on their official pages that LTSC is not meant to be used for normal users.

If i believed in what Microsoft says officially then i would be using Windows 10 Home edition on "Enhanced" telemetry level and playing UWP Minecraft on it, lmao. They say that only because they can't datamine LTSC users as much as they can datamine CB/CBB users, never believe in their marketing propaganda, you should learn to see marketing lies. 

Has Microsoft tech support ever helped anyone? I don't think so. 
99% of Windows problems have to be fixed by the dedicated community, Microsoft tech support employees are like bots, always spewing generic advices like sfc /scannow, so IT pros don't even take them seriously anymore, LMAO!

Honestly, i used to be a big fan of Microsoft and Windows when Steve Ballmer was still the CEO of MS, but then Satya Nadella came and ruined everything i loved Windows for. 
Windows 7 was great OS, in my humble opinion it's the best OS ever made, EVER. Aero Glass theme was beautiful to look at, everything worked as it should on 7  - i never ever had single problem with Win7. 
I would still use my Windows 7 Ultimate if it was supported.

**Comment by Atavic** [comment]
Both Firewall and Antivirus are surely better integrated if coming from MS and not 3rd parties.

**Comment by Drakonas** [comment]
In regards to telemetry, if you read Microsoft's privacy policy, it's not at all like Apple. They keep your data internal for the use of improving their services and your experience. They don't sell the information to ad partners at all. The only time it goes outside of the company is providing it to analysis partners for the purpose of improving their services, to which they are also under the same requirement to not sell or provide your data elsewhere, and then providing the data to the authorities in case there is a large-scale legal concern/warrant for your arrest.

We live in a day and age where you can't keep your data exluded from everyone. If you want to use the internet, you might as well face it: Your data is going to be used to improve the company's software. It's just how it is today. Why spend the manpower asking for feedback on software and pay out the wazzoo for more customer support when you can just create software to monitor and automatically improve aspects of the app based on the data it collects..

Yes, it's scary for a company to carry your info, and a lot of it, but it's just how the world of technology works today. There's nothing you can do about it other than just not using operating systems that collect data. (Which basically means using only Linux) As much as I love Linux, I'm a gamer. I'm not going to spend hours getting a game to run in Wine, when I can just use Windows and have everything work from the get-go.  If you are arguing against an operating system invading privacy, why are you still using it? In my experience, removing the telemetry broke a lot of software I wanted to use on Windows, as integral parts of Windows required those components to function properly (I still have some old friends on Skype that I don't want to lose touch with).

**Comment by CHEF-KOCH** [comment]
If someone likes to help me with [regtweaks project](https://github.com/CHEF-KOCH/regtweaks) we would have the biggest scripts and tweaks collection, otherwise, I have nothing to add in this conversation since we hitting off-topic.

**Comment by Atavic** [comment]
I'd like to help for documentation, if that means finding info and write comments about specific entries.

**Comment by YasserDRIF** [comment]
Hello there,

i know it's been about 2 years since the last update to this project
but i'm still thinking about posting a new version here
is anyone still interested in using this script and also is anyone interested in helping maintain it over the next period

@cryohellinc @Drakonas @Atavic @SceneAddict

**Comment by Drakonas** [comment]
I sadly can't be of any help in the coding, but I definitely would be very much interested in a new version of this! @YasserDRIF The extent of my coding knowledge is being able to hack up a script from multiple Stackoverflow listings. XD

**Comment by cryohellinc** [comment]
@YasserDRIF 
Please do! I still use it various PC's, your tool + BlackBird is really fantastic. (Any chance you can combine your work with their into something mega awesome?)

Maintainance wise, sadly I don't know any programming languages, but can gladly help test it.

Thank you for all the effort!

**Comment by Justaus3r** [comment]
@YasserDRIF 
Yeah please do ,i am interested in maintaining it but i can't promise it...
